packages/replayio/src/utils/browser/launchBrowser.ts

```diff
@@ -34,7 +33,7 @@ export async function launchBrowser(
       RECORD_ALL_CONTENT: "1",
       RECORD_REPLAY_DIRECTORY: getReplayPath(),
       RECORD_REPLAY_METADATA: JSON.stringify({ processGroupId }),
-      RECORD_REPLAY_VERBOSE: verbose ? "1" : undefined,
+      RECORD_REPLAY_VERBOSE: "1",
```

Comment:
This is a quick idea, the core of my change is in the playwright's package. It just seems that perhaps the same idea can just be used here - we enable verbose logs at all times **but** we still gate them behind `DEBUG`.

Note that I didn't test it out with the CLI yet. Rn I'm heading out but I wanted to open this to get the initial feedback.